bitcoin/src/lib.rs

```diff
@@ -139,6 +139,7 @@ pub use crate::hash_types::{
 };
 pub use crate::merkle_tree::MerkleBlock;
 pub use crate::network::constants::Network;
+pub use crate::p2p as network;
```

Comment:
This should be deprecated I guess?

Reply:
It is removed later in the PR, should have had a comment on it saying as such. I did this because later patches the final state of imports wasn't done till the end of the patch set so I didn't want to clutter the earlier patches with path changes that a) looked bad b) would be changed again.